apps/aecontract/src/aect_create_tx.erl

```diff
@@ -275,11 +275,11 @@ run_contract(#contract_create_tx{ owner      = Caller
 initialize_contract(#contract_create_tx{owner      = OwnerPubKey,
                                         nonce      = Nonce,
                                         vm_version = VmVersion,
-                                        amount     = _Amount,
-                                        gas        = _Gas,
+                                        amount     =_Amount,
```

Comment:
So the missing spaces are done on purpose? I should maybe stop "fixing" them then?!

Reply:
Yes, Tobias likes it that way, I am not so sure myself...